pkg/controller/zookeepercluster/zookeepercluster_controller.go

```diff
@@ -589,7 +589,7 @@ func (r *ReconcileZookeeperCluster) getPVCList(instance *zookeeperv1beta1.Zookee
 		LabelSelector: selector,
 	}
 	pvcList := &corev1.PersistentVolumeClaimList{}
-	err = r.client.List(context.TODO(), pvclistOps, pvcList)
+	err = r.client.List(context.TODO(), pvcList, pvclistOps)
```

Comment:
It's related to updating to gomodules.

Reply:
@ekarlso I think that change is needed because of change to the k8s version and not for go-modules. I'd prefer not conflating both changes in 1 PR, but instead to have 2 separate PRs one for the go modules and another for k8s version migration.